### DISCUSSION TITLE
Creating GitBash On MacBook

### DISCUSSION THREAD

**Discussion by JohnnyWhales** [learndevopsonline/learndevopsonline#1583] [Q&A] [answered]
Good Morning, I use a MacBook and i am trying to follow the training in cloning my repository on my MacBook but the video doesn't show how to do it on a man. Do you mind helping out on this. 

thanks you.

**Comment by sreedharm07** [comment]
Hi @JohnnyWhales 
 cloning my repository on my MacBook but the video **doesn't show how to do it on a **_man_**** means ? its a man or mac ?

**Reply by JohnnyWhales** [reply]
On a MacBook.

**Reply by sreedharm07** [reply]
Actually in training viedios raghu is using mac only.

**Reply by sreedharm07** [reply]
@JohnnyWhales 
how to clone repos from github to your mackbook? is this you are asking ?

**Reply by JohnnyWhales** [reply]
Yes, I know but during that training he used the Windows for the clone.

- How to Download gitbash, create gitbash repo on the device and cloning gitbash into IntelliJ. I have IntelliJ already downloaded. I ran the script from Gitbash but don't know where to go from there.

**Comment by JohnnyWhales** [comment]
<img width="1440" alt="snipeshot" src="https://github.com/learndevopsonline/learndevopsonline/assets/135075456/8bb6b516-d1bf-4a23-a4f6-bfeb206d8d6e">

Here is the snipe of the video.

**Reply by sreedharm07** [reply]
hi @JohnnyWhales 
1-download git from here -https://git-scm.com/downloads
2-create one folder for devops 
3-in command line go to that devops folder 
4-then give **git clone** _"git hub url"
5- now open your intilij and open devops folder which you created.
attached screenshot for your reference 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/2fb858df-97c1-4a37-b711-4d0d92d51ed7)

**Reply by sreedharm07** [reply, marked as answer]
in windows- will go to folder then git bash here and cloning repo
in mac - in your terminal go to folder and just do git clone

**Comment by JohnnyWhales** [comment]
Thank You everything is working now.
